### ISSUE TITLE
Doctest: Bio.ExPASy.get_prosite_raw Fails

### ISSUE THREAD

**Issue by svalqui** [biopython/biopython#3464]
### Setup

I am reporting a problem with Biopython version, Python version, and operating
system as follows:

3.6.9 (default, Oct  8 2020, 12:12:24) 
[GCC 8.4.0]
CPython
Linux-4.15.0-130-generic-x86_64-with-Ubuntu-18.04-bionic
1.79.dev0

Doctest: Bio.ExPASy.get_prosite_raw Fails when running 

`python3 setup.py test`

### Expected behaviour
The test to pass:
`Bio.ExPASy docstring test ... ok`

### Actual behaviour

(*Please fill this in, and provide any exception message in full*)

`Bio.ExPASy docstring test ... FAIL`

```
======================================================================
FAIL: get_prosite_raw (Bio.ExPASy)
Doctest: Bio.ExPASy.get_prosite_raw
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python3.6/doctest.py", line 2199, in runTest
    raise self.failureException(self.format_failure(new.getvalue()))
AssertionError: Failed doctest test for Bio.ExPASy.get_prosite_raw
  File "/home/Projects/biopython/build/lib.linux-x86_64-3.6/Bio/ExPASy/__init__.py", line 66, in get_prosite_raw

----------------------------------------------------------------------
File "/home/Projects/biopython/build/lib.linux-x86_64-3.6/Bio/ExPASy/__init__.py", line 82, in Bio.ExPASy.get_prosite_raw
Failed example:
    handle = ExPASy.get_prosite_raw("DOES_NOT_EXIST")
Expected:
    Traceback (most recent call last):
        ...
    ValueError: Failed to find entry 'DOES_NOT_EXIST' on ExPASy
Got nothing



----------------------------------------------------------------------
Ran 530 tests in 2614.157 seconds

FAILED (failures = 1)

```
### Steps to reproduce
Using the latest dev version 1.79.dev0

```
sudo python3 setup.py build
python3 setup.py test
```

**Comment by svalqui** [comment]
I'll have a look at this.

**Comment by svalqui** [comment]
@peterjc Could you please close this? I can't no longer reproduce this error, I might had something missing on one of my branches, Sorry.
`Bio.ExPASy docstring test ... ok`

**Comment by mdehoon** [comment]
Fixed by #3511
